src/include/rados/librados.h

```diff
@@ -2163,13 +2162,11 @@ CEPH_RADOS_API int rados_watch(rados_ioctx_t io, const char *o, uint64_t ver,
  * A watch operation registers the client as being interested in
  * notifications on an object. OSDs keep track of watches on
  * persistent storage, so they are preserved across cluster changes by
- * the normal recovery process. If the client loses its connection to
- * the primary OSD for a watched object, the watch will be removed
- * after 30 seconds. Watches are automatically reestablished when a new
+ * the normal recovery process. The timeout of the connection between
```

Comment:
can we keep the original doc, how about amending it like:

> If the client loses its connection to the primary OSD for a watched object, the watch will be removed after a timeout configured with `osd_client_watch_timeout`.

what do you think?